tstest/integration/testcontrol/testcontrol.go

```diff
@@ -67,6 +68,13 @@ func (s *Server) NumNodes() int {
 	return len(s.nodes)
 }
 
+// AddPingRequest adds the entry to the pingReqsToAdd map
+func (s *Server) AddPingRequest(nodeKeyDst tailcfg.NodeKey, pr *tailcfg.PingRequest) {
+	s.mu.Lock()
+	defer s.mu.Unlock()
+	s.pingReqsToAdd[nodeKeyDst] = pr
```

Comment:
you need to lazily initialize the map first if it's nil

Also, you'll need a `sendUpdate(....)` call in here too